digicampipe/visualization/mpl.py

```diff
@@ -1,584 +0,0 @@
-# Licensed under a 3-clause BSD style license - see LICENSE.rst
-"""
-Visualization routines using matplotlib
-"""
-import copy
-import logging
-
-import matplotlib
-import numpy as np
-from astropy import units as u
```

Comment:
This file should not be deleted in this PR

Reply:
Yes ... I messed up my master somehow ... and when I branched this branch off ... this modification was taken also into this branch. ... I have to fix this.